src/sound/customaudio.js

```diff
@@ -0,0 +1,82 @@
+const audioPlayer = document.querySelector('.audio-player-custom');
+const audio = new Audio();
+
+audio.addEventListener(
+	'loadeddata',
+	() => {
+		audioPlayer.querySelector('.time .length').textContent = getTimeCodeFromNum(audio.duration);
+		audio.volume = 0.75;
+	},
+	false,
+);
+
+const timeline = audioPlayer.querySelector('.timeline');
+timeline.addEventListener(
+	'click',
+	(e) => {
+		const timelineWidth = window.getComputedStyle(timeline).width;
+		const timeToSeek = (e.offsetX / parseInt(timelineWidth)) * audio.duration;
+		audio.currentTime = timeToSeek;
+	},
+	false,
+);
+
+const volumeSlider = audioPlayer.querySelector('.controls .volume-slider');
+volumeSlider.addEventListener(
+	'click',
+	(e) => {
+		const sliderWidth = window.getComputedStyle(volumeSlider).width;
+		const newVolume = e.offsetX / parseInt(sliderWidth);
+		audio.volume = newVolume;
+		audioPlayer.querySelector('.controls .volume-percentage').style.width = newVolume * 100 + '%';
+	},
+	false,
+);
+
+setInterval(() => {
+	const progressBar = audioPlayer.querySelector('.progress-music');
+	progressBar.style.width = (audio.currentTime / audio.duration) * 100 + '%';
+	audioPlayer.querySelector('.time .current').textContent = getTimeCodeFromNum(audio.currentTime);
+}, 500);
+
+const playBtn = audioPlayer.querySelector('.controls .toggle-play');
+playBtn.addEventListener(
+	'click',
+	() => {
+		if (audio.paused) {
+			playBtn.classList.remove('play');
+			playBtn.classList.add('pause');
+			audio.play();
+		} else {
+			playBtn.classList.remove('pause');
+			playBtn.classList.add('play');
+			audio.pause();
+		}
+	},
+	false,
+);
+
+audioPlayer.querySelector('.volume-button').addEventListener('click', () => {
+	const volumeEl = audioPlayer.querySelector('.volume-container .volume');
+	audio.muted = !audio.muted;
+	if (audio.muted) {
+		volumeEl.classList.remove('icono-volumeMedium');
```

Comment:
@Carlos-Carsdfj Oh, nvm the "icono" part, I see that's an actual pack named like that. We could use "icono-sync" from it https://saeedalipoor.github.io/icono/ which is also pure CSS icon instead of the game-icons.net stuff.